modules/webadmin.cpp

```diff
@@ -1145,10 +1221,12 @@ class CWebAdminMod : public CModule {
 
         VCString vsArgs;
 
-        pNetwork->DelServers();
-        WebSock.GetRawParam("servers").Split("\n", vsArgs);
-        for (const CString& sServer : vsArgs) {
-            pNetwork->AddServer(sServer.Trim_n());
+        if (spSession->IsAdmin() || !spSession->GetUser()->DenySetNetwork()) {
```

Comment:
This DenySetNetwork looks like a catch-all for all network-level settings, *and* adding new networks. What exactly it's supposed to achieve?

Reply:
It prevents adding and deleting networks, changing the network name and modifying the server list in any way